components/PicklistBase/PicklistBase.content.comp.cy.js

```diff
@@ -18,11 +18,230 @@ describe('Test the default PicklistBase content', () => {
       props: {
         onReady: readySpy,
         rows: [],
+        columns: ['name', 'quantity', 'location'],
+        labels: { name: 'Name', quantity: 'Quantity', location: 'Location' },
       },
     });
 
     cy.get('@readySpy')
       .should('have.been.calledOnce')
-      .then(() => {});
+      .then(() => {
+        cy.get('[data-cy="picklist-table"]').should('exist');
+        cy.get('[data-cy="picklist-header-name"]').should('exist');
+        cy.get('[data-cy="picklist-header-quantity"]').should('exist');
+        cy.get('[data-cy="picklist-header-location"]').should('exist');
+        cy.get('[data-cy="picklist-invalid-feedback"]').should('not.exist');
+        cy.get('[data-cy="picklist-sort-button-name"]').should('exist');
+        cy.get('[data-cy="picklist-sort-button-quantity"]').should('exist');
+        cy.get('[data-cy="picklist-sort-button-location"]').should('exist');
+      });
+  });
+
+  it('Check that props are passed through to the PicklistBase', () => {
+    const readySpy = cy.spy().as('readySpy');
+
+    cy.mount(PicklistBase, {
+      props: {
+        onReady: readySpy,
+        rows: [
+          { name: 'Item A', quantity: 1, location: 'GHANA' },
+          { name: 'Item B', quantity: 2, location: 'GHANA' },
+        ],
+        columns: ['name', 'quantity', 'location'],
+        labels: { name: 'Name', quantity: 'Quantity', location: 'Location' },
+        picked: [0, 0],
+        units: 'Trays',
+        quantityAttribute: 'quantity',
+        showValidityStyling: true,
+        invalidFeedbackText: 'Please pick at least one item.',
+        showInfoIcons: true,
+        required: true, // Ensuring required is true to trigger invalid state
+      },
+    });
+
+    cy.get('@readySpy')
+      .should('have.been.calledOnce')
+      .then(() => {
+        cy.get('[data-cy="picklist-table"]').should('exist');
```

Comment:
As above, this test should check the contents of the rows in the table.  Checking the first element of each row will be sufficient to ensure that the correct row of `rows` is being mapped to the correct row in the table.